code/datums/gamemode/role/role.dm

```diff
@@ -145,9 +145,6 @@
 		return 0
 
 	antag = M
-	if(M.current.client)
-		antag.key = M.current.client.ckey
-		antag.active = TRUE
```

Comment:
Removing the symptom treatment. The cause came from one_click_antag.dm's makeBody().